src/IntercomMessage.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace FtwSoft\NotificationChannels\Intercom;
+
+class IntercomMessage
+{
+    public const TYPE_EMAIL = 'email';
+
+    public const TYPE_INAPP = 'inapp';
+
+    public const TEMPLATE_PLAIN = 'plain';
+
+    public const TEMPLATE_PERSONAL = 'personal';
+
+    /**
+     * @param string $body
+     *
+     * @return \FtwSoft\NotificationChannels\Intercom\IntercomMessage
+     */
+    public static function create(?string $body = null): self
+    {
+        return new static($body);
+    }
+
+    /**
+     * @var array
+     */
+    public $payload;
+
+    /**
+     * IntercomMessage constructor.
+     *
+     * @param string $body
+     */
+    public function __construct(?string $body = null)
```

Comment:
The composer json lists the min version as `7.1` but that doesn't have nullable hints. Can you bump the min version in the composer.json to `7.2`?